projekt/src/styles.css

```diff
@@ -1,4 +1,46 @@
 /* You can add global styles to this file, and also import other style files */
 
-html, body { height: 100%; }
-body { margin: 0; font-family: Roboto, "Helvetica Neue", sans-serif; }
+html {    
+    font-family: 'Gill Sans', Arial, Helvetica, sans-serif;
+    font-size: 1em;
+    width: 100%;
+}
+
+.progress-bar {
+    width: 50%;
+    margin-left: 1em;
+    margin-bottom: .25em;
+        
+}
+
+.float-right {
+    float: right;
+    margin: 0 auto 0 auto;
```

Comment:
ln 18 : can be deleted, no use

Reply:
I deleted the class and will now merge the changes